README.rst

```diff
@@ -188,13 +188,186 @@ Triage is developed at the University of Chicago's `Center For Data Science and
 Major Components Used by Triage
 ===============================
 
-Triage makes use of many core data science components developed at DSaPP. These components can be useful in their own right, and are worth checking out if you'd like to make use of a subset of Triage's functionality in an existing pipeline:
+Triage makes use of many core data science components developed at DSaPP. These components can be useful in their own right, and are worth checking out if you'd like to make use of a subset of Triage's functionality in an existing pipeline.
+
```

Comment:
I'm thinking through the inclusion of the full catwalk README here when it's already in the catwalk directory. I like having more detail, than just listing 'Catwalk' with the link, but the Usage seems like a bit much.  The full introduction to catwalk seems a bit redundant (it was really written for people who had never seen triage).

Really, I think the content of most value here is probably the list of individual components with their one-liner descriptions.

Reply:
👍

Reply:
Should be good now

Reply:
Yeah I like this better as well